orchestra/utils/slack.py

```diff
@@ -0,0 +1,8 @@
+from django.conf import settings
+from slacker import Slacker
+
+
+def get_slack_user_id(slack_username):
+    slack = Slacker(settings.SLACK_EXPERTS_API_KEY)
```

Comment:
let's move this object into slack as well just using the `settings.SLACK_EXPERTS_API_KEY` as default